dashboard/app/helpers/aichat_safety_helper.rb

```diff
@@ -34,6 +63,14 @@ def find_toxicity(role, text, locale)
       DCDO.get("aichat_safety_webpurify_enabled_#{role}", false)
     end
 
+    private def openai_enabled?(role)
+      DCDO.get("aichat_safety_openai_enabled_#{role}", false)
```

Comment:
Setting the default for openai 'off' for now so @samantha-code can finish testing levels with aws comprehend and webpurify.
Note that on `production` and `levelbuilder` webpurify's DCDO updated to `true`.

Reply:
I'd suggest we commit with the defaults set how we anticipate going forward (with openai at this point), and set DCDO flags to override to let Sam keep working.

Reply:
We'd have to carefully monitor when DTP happens to set flags back and also make sure `production` and `levelbuilder` are in sync. So I think I prefer keeping as is if you're okay with that. Thanks! https://codedotorg.slack.com/archives/C07LQF3AGMA/p1727188797741739

Reply:
After discussion offline, we'll wait to merge this PR until after @samantha-code is done testing on `production` so setting default to OpenAI. Thanks!